nano/templates/ntnet_chat.tmpl

```diff
@@ -46,10 +49,13 @@
 			</div>
 		</div>
         </div>	
- 	<b>Connected Users</b><br>
-	{{for data.clients}}
-		{{:value.name}}<br>
-	{{/for}}
+	<div class="itemLabel">
+		Messaging: 
+	</div>
+	<div class="itemContent">
```

Comment:
You are opening a div here, but are not closing it

Reply:
Donezo.